src/components/Settings.jsx

```diff
@@ -86,6 +89,24 @@ export default function Settings({ currentWallet }) {
           />
           Use {settings.useAdvancedWalletMode ? 'magic' : 'advanced'} wallet mode
         </rb.Button>
+
+        <rb.Dropdown>
+          <rb.Dropdown.Toggle variant="outline-dark" className="border-0 mb-2 d-inline-flex align-items-center">
+            <Sprite symbol="globe" width="24" height="24" className="me-2" />
+            {languages.find((lng) => lng.key === i18n.language).description}
+          </rb.Dropdown.Toggle>
+
+          <rb.Dropdown.Menu variant={settings.theme === 'light' ? 'light' : 'dark'}>
+            {languages.map((lng, index) => {
+              return (
+                <rb.Dropdown.Item key={index} onClick={() => i18n.changeLanguage(lng.key)}>
+                  {lng.description}
+                </rb.Dropdown.Item>
+              )
+            })}
+            <rb.Dropdown.Item href="https://t.me/JoinMarketWebUI">Missing your language? Help us out!</rb.Dropdown.Item>
```

Comment:
Really great idea to add a link here :muscle:

First I thought it might be good to link to the nicely written i18n readme, as it also contact information.. but I guess we want to interact directly with everyone trying to help adding a new language?

Reply:
Oh yeah no you're right, the I'll link to the readme that's better.